chain/mining.go

```diff
@@ -47,14 +47,16 @@ func (bm *BuiltinMining) BuildBlock(height uint32, chordID []byte, winningHash c
 	txCount := 1
 
 	if winnerType == TXN_SIGNER {
-		miningSigChainTxn, err := por.GetPorServer().GetMiningSigChainTxn(winningHash)
-		if err != nil {
-			return nil, err
+		if _, err = DefaultLedger.Store.GetTransaction(winnerHash); err != nil {
```

Comment:
What happen if the winnerHash was in ledger already?
It seems that it will build a block without any sigChains in this proposal.

Reply:
Yes, but it will not cause any problem.